src/editor/styles.less

```diff
@@ -0,0 +1,37 @@
+.ui.popup.high {
```

Comment:
@avoinea How about making these styles configurable? For now these are taken from https://github.com/eea/volto-eea-design-system/blob/develop/theme/themes/eea/elements/label.overrides#L10